packages/cli/lib/cli.ts

```diff
@@ -1,31 +1,33 @@
 import fs from 'node:fs';
 import path, { dirname } from 'node:path';
 import { fileURLToPath } from 'node:url';
+
 import chalk from 'chalk';
 import chokidar from 'chokidar';
-import ejs from 'ejs';
 import * as dotenv from 'dotenv';
+import ejs from 'ejs';
+
+import { getProviderConfigurationFromPath, nangoConfigFile } from '@nangohq/nango-yaml';
 
-import { getNangoRootPath, printDebug } from './utils.js';
-import { loadYamlAndGenerate } from './services/model.service.js';
 import { TYPES_FILE_NAME, exampleSyncName } from './constants.js';
 import { compileAllFiles, compileSingleFile, getFileToCompile } from './services/compile.service.js';
+import { loadYamlAndGenerate } from './services/model.service.js';
 import { getLayoutMode } from './utils/layoutMode.js';
-import { getProviderConfigurationFromPath, nangoConfigFile } from '@nangohq/nango-yaml';
+import { getNangoRootPath, printDebug } from './utils.js';
 import { NANGO_VERSION } from './version.js';
 
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = dirname(__filename);
 
 dotenv.config();
 
-export const version = (debug: boolean) => {
+export const getVersionOutput = (debug: boolean): string => {
     if (debug) {
         printDebug('Looking up the version first for a local path first then globally');
     }
```

Comment:
I forgot to remove this when I put the version in a file, can you remove that and the debug flag? ☺️